src/reference-manual/types.Rmd

```diff
@@ -1043,9 +1068,9 @@ follows the assignment rules).
 For example, vectors cannot be assigned to arrays or vice-versa.
 
 ```stan
-real a[4];
+array[4] real a;
 vector[4] b;
-row_vector c[4];
+array[4] row_vector c;
```

Comment:
I think the source statement is wrong. Should be `row_vector[4] c`. Not sure what's going on here.